www/include/configuration/configObject/service/listServiceByHost.php

```diff
@@ -84,15 +84,15 @@
     $centreon->historySearch[$url]["searchS"] = $searchS;
     isset($_POST["statusHostFilter"]) ? $hostStatus = 1 : $hostStatus = 0;
     $centreon->historySearch[$url]["hostStatus"] = $hostStatus;
-} elseif (isset($_GET['Search'])) {
+} elseif (isset($_GET['search'])) {
```

Comment:
as you can see $_GET['search'] and $_POST['Search'] do the same thing. why do you keep duplicated code, when you could modify line 75 and remove the elseif ?

Reply:
you didn't mofidied the line 75 : $_POST['Search'] -> $_POST['search']
(how can it work on your env ??)